src/assets/tailwind.css

```
@@ -28,6 +28,20 @@ div[contenteditable='true']:focus {
   grid-template-rows: auto auto minmax(0, 1fr) auto minmax(0, 1fr);
 }
 
+.column-resizer {
+  position: relative;
```

Since we are using tailwind (in the form of windicss but that's all the same), we might as well use the little helpers it provide for cleaner code. This also reduce the conflicting question of what z-index to pick. z-50 is the highest value on the tailwind scale

```suggestion
  @apply relative w-50;
```